contentcuration/contentcuration/frontend/channelEdit/views/files/FileUploadItem.vue

```diff
@@ -133,6 +134,13 @@
         }
         return this.file;
       },
+      formattedFileDisplay() {
+        const fileName = this.fileDisplay.original_filename;
+        if (fileName === 'file' || (fileName && !fileName.length)) {
```

Comment:
It doesn't look like `fileName` would ever be `null` and in JS an empty string would be falsy so I don't think this will handle the case of empty strings. I would suggest changing:
```js
(fileName && !fileName.length)
```
to:
```js
!fileName
```
Here's how I checked that:
```
$ node
Welcome to Node.js v16.17.0.
Type ".help" for more information.
> var x = "";
undefined
> Boolean(x && !x.length)
false
> !x
true
```

Reply:
@bjester, my app=ologies, so there seems to be a disconnect between what was actually done and the title of the pull request. We decided not to pursue the idea of making the string null, because we would now need to handle two scenarios at the front-end ie all files with the name "file" and the new ones with the empty name(''). Thus explains why the implementation is based on the string 'file' and not the empty string cc @rtibbles

Reply:
Okay so no worries about changing the `file` string on the backend. As it exists in the PR, the bit of code I mentioned, `(fileName && !fileName.length)` should be removed if you don't want to handle empty strings, or otherwise corrected as I mentioned in the previous comment. 

Since our data model supports saving empty strings to the database, I think handling an empty string on the frontend does align with that. It also makes the data model the source of truth. I checked a production backup of the database and it does seem that there are files that have an empty string `original_filename`. An alternative is to enforce that file never has an empty string for this through a database constraint, which would require a different update.